src/Controllers/DevCheckController.php

```diff
@@ -43,8 +43,11 @@ public function index($request)
             $suite = $name;
         }
 
-        $checker = new EnvironmentChecker($suite, 'Environment status');
-        $checker->setRequest($request);
+        /** @var EnvironmentChecker */
+        $checker = EnvironmentChecker::create($suite, 'Environment status')
+            ->setRequest($request)
+            ->setIncludeDetails(false);
```

Comment:
```suggestion
            ->setIncludeDetails(true);
```
This is the controller that _should_ include details. It's already false by default for the `DevHealthController`'s sake.